github-diff.js

```diff
@@ -0,0 +1,141 @@
+const ADD = 'addition';
+const DELETE = 'deletion';
+const NONE = 'none';
+
+class DiffLine {
+  constructor(fileName, fileAnchor, lineNumber, type, text) {
+    this.fileName = fileName;
+    this.fileAnchor = fileAnchor;
+    this.lineNumber = lineNumber;
+    this.type = type;
+    this.text = text.replace(/ /g, "&nbsp;");
+  }
+
+  renderHtml() {
+    if (this.type === ADD) {
+      return `
+        <tr>
+          <td class="blob-num blob-num-addition empty-cell"></td>
```

Comment:
wondering how stable these class names are; Github tends to (or tended to when i was originally developing this extension) change their class names far more frequently than i thought they would